### DISCUSSION TITLE
Running queries on setInterval

### DISCUSSION THREAD

**Discussion by PixelsCommander** [blitz-js/blitz#1449] [Q&A] [answered]
Hi

I want to run queries on setInterval to refresh page content. And since queries require hooks I can't really put them inside of setInterval callback.

In version 0.22 it was possible to call queries directly as promises without specifying the context but in 0.24 this seems to be not possible so running queries requires useQuery therefore I cant run them to refresh page.

Any ideas on the solution?

**Comment by flybayer** [comment, marked as answer]
Did you try the `refetchInterval` for `useQuery`?

https://blitzjs.com/docs/use-query

**Reply by PixelsCommander** [reply]
Ugh, sorry. So obvious. I was not even thinking of this as a common problem solved in this way.  Thanks!
